folly/portability/Malloc.h

```diff
@@ -28,8 +28,12 @@
 #endif
 // JEMalloc provides it's own implementation of
 // malloc_usable_size, and that's what we should be using.
+#ifndef __FreeBSD__
```

Comment:
I prefer `#if !defined(...)`, since that way it is easier to add clauses to the condition as necessary.